client/src/components/AppFrame/AppFrame.js

```diff
@@ -92,7 +98,9 @@ const AppFrame = (props) => ({
       )
     } else if(currentView === 'imageScrubber') {
       tabContents = (
-        <ImageScrubber />
+        <Infinite containerHeight={scroll.containerHeight} elementHeight={scroll.elementHeight}>
```

Comment:
Not necessary, but out of curiosity can you pass it with the spread right? `<Infinite {...scroll}>